boards/matek/h743/default.px4board

```diff
@@ -49,15 +50,16 @@ CONFIG_MODULES_MC_RATE_CONTROL=y
 CONFIG_MODULES_NAVIGATOR=y
 CONFIG_MODULES_RC_UPDATE=y
 CONFIG_MODULES_SENSORS=y
-CONFIG_MODULES_TEMPERATURE_COMPENSATION=y
-CONFIG_MODULES_VTOL_ATT_CONTROL=y
```

Comment:
I could see people using this board for VTOL, is there not enough flash?

Reply:
VTOL is added but CAN interface is removed 
```
[896/899] Linking CXX executable matek_h743_default.elf
Memory region         Used Size  Region Size  %age Used
            itcm:          0 GB        64 KB      0.00%
           flash:     1581481 B      1920 KB     80.44%
           dtcm1:          0 GB        64 KB      0.00%
           dtcm2:          0 GB        64 KB      0.00%
            sram:       47236 B       512 KB      9.01%
           sram1:          0 GB       128 KB      0.00%
           sram2:          0 GB       128 KB      0.00%
           sram3:          0 GB        32 KB      0.00%
           sram4:          0 GB        64 KB      0.00%
           bbram:          0 GB         4 KB      0.00%
[898/899] uploading px4
```

if not removed 
```
ERROR: Firmware image is too large for this board
```

Reply:
Ok I'll take a closer look at the flash usage once all of this settles in master. We'll have some significant flash savings soon.